docker/sconify.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# IMG_FROM=iexechub/iexec-sms:7.1.0 IMG_TO=iexechub/iexec-sms:7.1.0-debug ./sconify.sh
+cd $(dirname $0)
+
+SCONE_IMG_NAME=sconecuratedimages/iexec-sconify-image
+SCONE_IMG_VERSION=5.7.0-wal
+
+IMG_TO=${IMG_TO}-sconify-${SCONE_IMG_VERSION}-debug
+
+ARGS=$(sed -e "s'\${IMG_FROM}'${IMG_FROM}'" -e "s'\${IMG_TO}'${IMG_TO}'" sconify.args)
+echo $ARGS
+
+SCONE_IMAGE="registry.scontain.com:5050/${SCONE_IMG_NAME}:${SCONE_IMG_VERSION}"
+
+/bin/bash -c "docker run -t --rm \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    ${SCONE_IMAGE} \
+        sconify_iexec \
+            --cli=${SCONE_IMAGE} \
+            --crosscompiler=${SCONE_IMAGE} \
+            $ARGS"
+
+echo
+docker run --rm -e SCONE_HASH=1 $IMG_TO
```

Comment:
I think this line should feature the heapsize too

Reply:
Do you mean:
1. It is mandatory, we should add it
or
2. It would be great to ensure we build with the right heap size
?

Reply:
I see `--heap="8G"` set during build time, but I don't see it when `mrenclave` is printed here L24
"some of the [SCONE environment](https://sconedocs.github.io/SCONE_ENV/) variables like `SCONE_HEAP` and `SCONE_ALLOW_DLOPEN` will affect MrEnclave." (https://sconedocs.github.io/MrEnclave/)

Reply:
Sure, but this doesn't seem to be mandatory there. Example:
```
$ docker run --rm -e SCONE_HASH=1 docker-regis.iex.ec/iexec-sms:feature-ce6cd412-sconify-5.7.0-wal-debug
56453857e497e1b169acc36416f076225cdbf2fe7fe10f5976e1ec22a2dc63fa
$ docker run --rm -e SCONE_HASH=1 -e heap=8G docker-regis.iex.ec/iexec-sms:feature-ce6cd412-sconify-5.7.0-wal-debug
56453857e497e1b169acc36416f076225cdbf2fe7fe10f5976e1ec22a2dc63fa
```

Reply:
By default, a sconified image seems to provide a mrenclave based on provided sconify args.
It is interesting to know if we want to modify heap size without sconifying the application.
But it means we have to define SCONE_HEAP when creating the session (build enclave config mechanism)